### DISCUSSION TITLE
Build for Android?

### DISCUSSION THREAD

**Discussion by kdpuvvadi** [Jacalz/rymdport#46] [Q&A] [answered]
### Describe the solution you'd like to see:

is there any plan for releasing a build for android?

**Comment by Jacalz** [comment, marked as answer]
There certainly is :)
I have a (albeit ageing and not complete) implementation in the [mobile3](https://github.com/Jacalz/rymdport/tree/mobile3) branch. I plan on doing some small improvement soon that should make the mobile support easier to implement in the future.

It has unfortunately had to be prioritised down due to lack of time (I am studying Computer Science and Engineering at the university) and [lack of funding](https://github.com/Jacalz/rymdport/tree/main#sponsoring) (basically that I don't have any Android phone and the marketplaces cost money to set up and maintain). But yes, the plan is definitely to support Android and other mobile platforms.

However, I will go ahead and close this as a duplicate of https://github.com/Jacalz/rymdport/issues/19.

**Reply by Jacalz** [reply]
For anyone interested, I just want to share that https://play.google.com/store/apps/details?id=eu.heili.wormhole is avaliable as a wormhole client on Android. Anyone that wants to install that should be able to send and receive items between that and Rymdport. I will try to get QR code generation into the next release of Rymdport to make it easier :)
